packages/rocketchat-livechat/.app/client/views/switchDepartment.js

```diff
@@ -36,40 +36,36 @@ Template.switchDepartment.events({
 
 		instance.error.set();
 		swal({
-			text: t('Are_you_sure_do_you_want_end_this_chat_and_switch_department'),
+			text: t('Are_you_sure_do_you_want_switch_the_department'),
 			title: '',
 			type: 'warning',
 			showCancelButton: true,
 			confirmButtonColor: '#DD6B55',
 			confirmButtonText: t('Yes'),
 			cancelButtonText: t('No'),
 			html: false
-		}).then((result) => {
-			if (!result.value) {
-				return;
-			}	
+		}, () => {
```

Comment:
I think we updated the sweetalert and now its promise based, we should change to 
```javascript
}).then(() => {
```